src/App.tsx

```diff
@@ -1,14 +1,74 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { TodoList } from './Components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { Footer } from './Components/Footer/Footer';
+import { StatesOfFilter } from './types/StatesOfFilter';
+
+function getFilteredTodos(todos: Todo[], filterOption: StatesOfFilter): Todo[] {
+  let filteredTodos = [...todos];
```

Comment:
- Move to the utils/helpers folder/file.
- You don't need to create a copy as `filter` doesn't mutate your array